src/components/atoms/Button/Button.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react'
 import styled from 'styled-components'
 
+import { DefaultComponentProps } from '@types'
+
 // A T O M S
 import { Icon, IconName } from '../Icon/Icon'
 
 // I N T E R F A C E
-export interface ButtonProps {
-  className?: string
+export interface ButtonProps extends DefaultComponentProps {
```

Comment:
Naise 👍 we should add this to the documentation at some point

Reply:
Yes I guess it now make sense to introduce a `CONTRIBUTING.md` file 👍